unlock-app/src/components/interface/GlobalWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react'
+import GlobalStyle from '../../theme/globalStyle'
+import configure from '../../config'
+import { WedlockServiceContext } from '../../contexts/WedlocksContext'
+import WedlockService from '../../services/wedlockService'
+import { ConfigContext } from '../../utils/withConfig'
+import ProviderContext from '../../contexts/ProviderContext'
+import Authenticate from './Authenticate'
+import { CONSOLE_MESSAGE } from '../../constants'
+
+const config = configure()
+const wedlockService = new WedlockService(config.services.wedlocks.host)
+
+interface GlobalWrapperProps {
+  children: React.ReactNode
+  pageProps: any
+}
+
+export const GlobalWrapper = ({ children, pageProps }: GlobalWrapperProps) => {
+  const [provider, setProvider] = useState<any>(null)
+  useEffect(() => {
+    /* eslint-disable no-console */
+    console.info(CONSOLE_MESSAGE)
+    /* eslint-enable no-console */
+  }, [])
+
+  return (
+    children && (
+      <>
+        <GlobalStyle />
+        <ConfigContext.Provider value={config}>
+          <WedlockServiceContext.Provider value={wedlockService}>
+            <ProviderContext.Provider value={{ provider, setProvider }}>
```

Comment:
I can't tell what this Provider is for (I had to look up the hook file). It would be good to have a more descriptive name here but out of context for this PR.

Reply:
Agreed!